src/cascadia/TerminalSettings/TerminalSettings.cpp

```diff
@@ -149,6 +150,16 @@ namespace winrt::Microsoft::Terminal::Settings::implementation
         _snapOnInput = value;
     }
 
+    bool TerminalSettings::AltGrAliasing() noexcept

```

Comment:
Heads up, this will inevitably conflict with #6110, but that's not too bad to resolve